CONTRIBUTING.md

```diff
@@ -125,16 +125,51 @@ melos run firebase:emulator
 
 To run tests against Android, iOS or macOS environments, run these commands:
 
+For the `cloud_firestore` plugin:
 ```bash
-cd packages/firebase_auth/firebase_auth/example
-flutter test integration_test/firebase_auth_e2e_test.dart
+cd packages/cloud_firestore/cloud_firestore/example
+flutter test integration_test/cloud_firestore_e2e_test.dart
 ```
 
-To run tests against web environments, run the command as a release build:
+For the `firebase_performance` plugin:
+```bash
+cd packages/firebase_performance/firebase_performance/example
+flutter test integration_test/firebase_performance_e2e_test.dart
+```
 
+For the rest of the plugins:
 ```bash
-cd packages/firebase_auth/firebase_auth/example
-flutter drive --target=./integration_test/firebase_auth_e2e_test.dart --driver=./test_driver/integration_test.dart --release -d chrome
+cd tests
+flutter test integration_test/e2e_test.dart
+```
```

Comment:
Yes probably worth removing and use melos for everything 👍 
We could probably also add in another PR chromedriver to the melos command so everything is running in 1 command?